examples/sdk-examples/src/examples/timeOverTimeComparison/PreviousPeriodHeadlineExample.tsx

```diff
@@ -10,37 +10,35 @@ const TotalSales = modifyMeasure(Md.$TotalSales, (m) =>
     m.format("#,##0").alias("$ Total Sales").title("Total Sales"),
 );
 
-export class PreviousPeriodHeadlineExample extends Component {
-    public onLoadingChanged: OnLoadingChanged = (...params) => {
+export const PreviousPeriodHeadlineExample: React.FC = () => {
+    const onLoadingChanged: OnLoadingChanged = (...params) => {
         // eslint-disable-next-line no-console
         return console.log("PreviousPeriodHeadlineExample onLoadingChanged", ...params);
     };
 
-    public onError: OnError = (...params): void => {
+    const onError: OnError = (...params): void => {
         // eslint-disable-next-line no-console
         return console.log("PreviousPeriodHeadlineExample onError", ...params);
     };
 
-    public render() {
-        const primaryMeasure = TotalSales;
-        const secondaryMeasure = newPreviousPeriodMeasure(
-            TotalSales,
-            [{ dataSet: Md.DateDatasets.Date.identifier, periodsAgo: 1 }],
-            (m) => m.alias("$ Total Sales - period ago"),
-        );
+    const primaryMeasure = TotalSales;
```

Comment:
If these do not need anything from the state, we should put them outside of the component.